equivalent tab bar styling example with current api

saep opened the pull request:
Fixes #5219 

I've also been a little confused when I tried out the example from the docs. So, I've added an equivalent definition for the tab bar that works for the current version.

wez commented:
Thanks for this!

If the example in the docs doesn't work for the currently released version, please update the PR to update the example rather than adding a new one; that will help keep things less confusing overall in the docs.

saep commented:
I've moved the example to the top, so that it is outside the versioned sections and I've deleted the old one.

I've also tested the code again on version `20240203-110809-5046fc22` (relatively current nixos-unstable version).
